src/org/labkey/targetedms/parser/SkylineDocumentParser.java

```diff
@@ -183,9 +183,10 @@
 
     /**
      * To prevent giant DIA documents from overwhelming the DB, we skip importing TransitionChromInfos if the document
-     * has more than 100,000
+     * has more than 100,000 AND has more than 1,000 precursors
```

Comment:
```suggestion
     * has more than 100,000 OR has more than 1,000 precursors
```

Reply:
The text is accurate. It's describing the opposite behavior of the method below, hence the < comparison and || in the code.